### ISSUE TITLE
Show succession of points

### ISSUE THREAD

**Issue by arkhi** [arkhi/breadcrumbs#1]
@sylvainv mentioned adding numbers on each corner of the polyline would help to visualize the trail.

This or a fading polyline…

**Comment by sylvainv** [comment]
No, I mentionned removing the polylines all completely, but actually fading it or dotting and fading it would be better, just to help people find the next point.

**Comment by arkhi** [comment]
Leaving points only is unlikely to help people finding next point, so I’ll make some tests with the ideas in the previous comments…

Thanks for the precision. :)
